README.md

```diff
@@ -1,82 +1,125 @@
+
+To configure the Platform9 Express CLI to communicate with the Platform9 management plane, run the following command:
+
+```
+./pf9-express -s
+```
+
+Example:
 
 ```
 # ./pf9-express -s
 NOTE: to enter a NULL value for prompt, enter '-'
- 
+
 PF9 Management Plane URL [https://company.platform9.net]:
 --> accepted: https://company.platform9.net
- 
+
 Admin Username [user@company.com]:
 --> accepted: user@company.com
- 
+
 Admin Password [********]:
 --> accepted: ********
- 
+
 Region [Sunnyvale]:
 --> accepted: Sunnyvale
- 
+
 Tenant [service]:
 --> accepted: service
- 
+
 Manage Hostname [true false] [false]:
 --> accepted: false
- 
+
 Manage DNS Resolver [true false] [false]:
 --> accepted: false
- 
+
 DNS Resolver 1 [8.8.8.8]:
 --> accepted: 8.8.8.8
- 
+
 DNS Resolver 2 [8.8.4.4]:
 --> accepted: 8.8.4.4
- 
+
 Proxy URL:
 --> accepted: -
 ```
 
 ## Install Prerequisite Packages
-To install prerequisite packages on the Platform9 Express control host, run the following command (a sample session is included):
+
+To install prerequisite packages on the Platform9 Express control host, run the following command:
+
+```
+./pf9-express -i
+```
+
+Example:
 
 ```
 # ./pf9-express -i
 --> Installation Log: ./log/pf9-express.2018-05-22_11:36:13.log
 --> Validating package dependencies: epel-release ntp nginx gcc python-devel python2-pip bc shade docker-py ansible
 ```
 
-## Configuration Inventory (CLI Only)
-Platform9 Express uses Ansible to execute commands on the hosts to be taken under management.  In order to configure Ansible to run remote commands on the managed hosts, the Ansible Inventory file must be configured.  This file is located in /opt/pf9-express/inventory/hosts.
+## Configuring the Inventory (CLI Only)
+
+Platform9 Express uses Ansible to execute commands on the hosts to be taken under management.  In order to configure Ansible to run remote commands on the managed hosts, the Ansible Inventory file must be configured.  This file is located in **/opt/pf9-express/inventory/hosts**.
 
-NOTE: A sample template is installed in the previous command ("./pf9-express -s"). A breakdown of the Inventory File is below:
+> Platform9 Express supports Ansible's `group_vars` and `host_vars` methods of defining variables.
+
+A sample template is installed in the setup command (**./pf9-express -s**). A breakdown of the inventory file is below:
 
 ## Sample Inventory File Part 1 - Authentication Portion
-This is where you enter the credentials for your control host to log into the target VM hosts to be managed by the Platform9 management plane (through either a password or SSH key, comment out any password lines if using SSH authentication and vice versa as needed)
+
+This is where you enter the credentials for your control host to log into the target hosts to be managed by the Platform9 management plane.
+
+> When using password authentication, comment out `ansible_ssh_private_key_file`. When using a private key, comment out `ansible_sudo_pass`.
```

Comment:
Private key portion should be comment out `ansible_ssh_pass`

Reply:
Fixed.